templates/webapi/admin/user/index.html.ep

```diff
@@ -9,6 +9,14 @@
     <div class="col-sm-12">
         <h2><%= title %></h2>
 
+        <div id="error" class="alert alert-danger" role="alert" style="display:none">
```

Comment:
The generic way this error is handled means that this is not necessarily an "Internal server error". Either distinguish the errors or keep it simple, e.g. "An error occurred when changing the user role.". I also wouldn't include the link to the bug tracker. After all the server might just be not reachable because the client's internet connection went off. And I would prefer to simply show a flash message using `addFlash('danger', …);` instead of creating the elements manually.

Reply:
Good point. I didn't have notice of addFlash, very useful